install.sh

```diff
@@ -63,7 +63,7 @@ function main() {
   }
   local key val
   for key in $( echo "$json" | jq -r 'keys[]' ) ; do
-    val=$( echo "$json" | jq -r .$key )
+    val=$( echo "$json" | jq .$key )
     if [[ $val == true ]] || [[ $val == false ]] || [[ $val =~ ^[0-9]+$ ]] ; then
```

Comment:
Do you still need this here then?

Reply:
Yeah, because json supports ints and booleans "natively", which means you can have"
```
{ "key": true }
```
or
```
{ "key": 123 }
```
which `jq` will yield unquoted, which is explicitly problem for booleans (we need them as strings, not as bools) and sometimes for integers as well (`60.0`).

Reply:
Yeah I know, I thought you're forcing quoting anyway, but I didn't know how the input looks like.

Reply:
Forcing through `echo` wouldn't not work, because:
```
echo '{"key":"quo\"te"}' | jq -r .key
quo"te
```
which breaks the value even more.